### PR TITLE
[FIX] l10n_ar_account_tax_settlement: Mejoramos mensaje de error

### PR DESCRIPTION
**Pull request by mem-adhoc**
Ticket: 75353

### CODE REVIEW

**Comment by roboadhoc** [comment]
[Pull request status dashboard](https://mergebot.adhoc.ar/ingadhoc/odoo-argentina-ee/pull/400).

File: l10n_ar_account_tax_settlement/wizards/inflation_adjustment.py [hunk 1]
```diff
@@ -233,8 +233,7 @@ def FormatAmount(amount):
 
         if not lines:
             raise UserError(_(
-                "No hemos encontrado ningún asiento contable asociado al"
-                " periodo seleccionado."
+                "No hemos encontrado ningún asiento contable para ajustar. "
```
Comment (zaoral) [inline l10n_ar_account_tax_settlement/wizards/inflation_adjustment.py]: para el periodo seleccionado

**Comment by pablohmontenegro** [comment]
@roboadhoc r+

**Comment by zaoral** [comment]
@roboadhoc override=ci/runbot-oba

we force it because we are having a problem with runbot that the build is not linked

**Pull request closed without merging**